cerberus/tests/conftest.py

```diff
@@ -62,6 +63,13 @@ def validator():
         'min': 1,
         'max': 100,
     },
+    'a_decimal': {
+        'type': 'decimal',
+    },
+    'a_decimal_coercing': {
```

Comment:
this schema is only used for validation so far. it stems back from a time when Cerberus was far less complex. putting more complex stuff here has a chance to break a lot of tests.
there's a test module for normalizations you propably have noticed when you studied the code.